src/common/Common.h

```diff
@@ -41,6 +41,9 @@
 
 #define MAX_NETCLIENT_PACKET_SIZE (32767 - 1)               // Client hardcap: int16 with trailing zero space otherwise crash on memory free
 
+inline unsigned long atoul(char const* str) { return strtoul(str, nullptr, 10); }
```

Comment:
No no no, please

Reply:
What do you suggest instead? I didn't quite understand this part so I am more than happy to change and learn =)

Reply:
Maybe I just convert the string to uint32 and not unsigned long (long)

Reply:
Or I can use Acore::StringTo<uint32>? Will test that